blogposts/structural-code-search-at-scale-from-your-browser.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Structural code search. At scale and from your browser.
+author: Rijnard van Tonder
+authorUrl: https://twitter.com/rvtond
+publishDate: 2020-01-24T10:00-07:00
+tags: [blog]
+slug: structural-code-search-at-scale-from-your-browser
+heroImage: https://TODO
```

Comment:
FIXME

Reply:
You could this for `heroImage`:

<img width="815" alt="Screen Shot 2020-02-01 at 10 12 44 AM" src="https://user-images.githubusercontent.com/133014/73582993-8fd74d80-44db-11ea-8944-a7b719e82187.png">

And this for the hero image at the top of the post.

<img width="1360" alt="Screen Shot 2020-02-01 at 10 11 16 AM" src="https://user-images.githubusercontent.com/133014/73583075-ee043080-44db-11ea-8a9e-74454e31c70c.png">

Reply:
@rvantonder Do you want me to add these images to the post?

Reply:
Appreciate the offer! I like the idea of using a screenie of search results. It'd be cool if we could get a screenie with multiple line matches, so I think I can take care of this.